src/midi/midi_mt32.h

```diff
@@ -45,16 +45,16 @@ class MidiHandler_mt32 final : public MidiHandler {
 	static constexpr int FRAMES_PER_BUFFER = 1024; // synth granularity
 	static constexpr int SAMPLES_PER_BUFFER = FRAMES_PER_BUFFER * 2; // L & R
```

Comment:
These static constexpr class members can now be hidden in cpp file, they no longer need to be exposed in header file, I believe?

Reply:
Yes; another nice catch @dreamer!
Moved them out of the header